### ISSUE TITLE
Why we should use normalize to the image?

### ISSUE THREAD

**Issue by xiazhi1** [yjh0410/new-YOLOv1_PyTorch#10]
Hi! Thanks for awesome work!
Recently, I am trying to design a new object detection network based on yolov1, my work is based on another work https://github.com/abeardear/pytorch-YOLO-v1/blob/master/train.py . I have tested that my work can overfit in Pascal VOC 2007+2012 trainval. But my work suffered heavy overfit . I have tried dropout weight decay and data augment . But the problem is still not solved. And I noticed your work , I find that you will do a normalize to image which is other work doesn't have . Why you do it? Is is have an advantage in overcoming overfit?
Looking forward to your reply!
